tests/test_order.py

```diff
@@ -97,6 +98,31 @@ def test_order_parameters(order_mock):
     assert not order_mock.order_parameters
 
 
+def test_get_assets(order_mock, monkeypatch):
```

Comment:
I use Order_mock here despite we would like to move out from using mocks, because I am restoring previous tests that existed before we removed the function. Refactoring Order class and testing can be taken in a later task.